src/main/java/com/feedzai/commons/sql/abstraction/engine/configuration/PdbProperties.java

```diff
@@ -150,6 +152,14 @@
      * space in the DB server. This depends on the database implementation.
      */
     public static final String DISABLE_LOB_CACHING = "pdb.disable_lob_caching";
+    /**
+     * Property that indicates the time of login timeout in the socket.
+     */
+    public static final String LOGIN_TIMEOUT = "pdb.login_timeout";
+    /**
+     * Property that indicates the time of login timeout in the socket.
```

Comment:
Copy pastey from above.

Reply:
You are right. My bad. Thanks for mention it.